.github/gh-actions-self-hosted-runners/self-hosted-windows/startupScript.ps1

```diff
@@ -16,13 +16,22 @@
 #     limitations under the License.
 # -->
 
+Start-Sleep -Seconds 60
 $env:Path += ';C:\Program Files\git\bin' 
-  
-
 $response= Invoke-RestMethod https://api.github.com/repos/actions/runner/tags
 $version= $response[0].name.substring(1,$response[0].name.Length-1)
+
+$ORG_NAME="apache"
+$ORG_RUNNER_GROUP="beam"
```

Comment:
I think it should be "Beam"

Reply:
Yep, thx , I've changed the name to Beam